usgs-details.html

```diff
@@ -239,14 +292,21 @@
             xhr.send();
         }
 
+
+
         // Get metric from Query Parameter and fetch and populate values
         window.onload = function() {
             var metric = METRICS[getQueryParameterByName('metric')];
             if (!metric) {
                 console.warn('Could not find metric for ' + metric);
+                
+                // Sensor lacks metrics, so giving its div.container a name of "sensor"
+                // in order to apply some specific styles.
+                var container = document.getElementsByClassName('container');
+                container[0].setAttribute('name', 'sensor');
```

Comment:
This seems like it works well! I was able to load and unload the different spokes and this shows up only when the sensor spoke's selected. 

I wonder if it'd make sense to give the `container` div above a unique string like `sensor-data-container` for its `id` attribute so that we could use something like

```js
var container = documentGetElementById('sensor-data-container');
container.setAttribute('name', 'sensor');
```

to get the single element and then not have to use the array index here.